.travis.yml

```diff
@@ -9,42 +9,80 @@
 ##
 
 sudo: required
+
 dist: trusty
+
 branches:
   only:
     - master
-matrix:
-  fast_finish: true
 
-env:
-  - TARGET="Style"
-  - TARGET="Registers"
+matrix:
+  include:
+    - os: osx
+      osx_image: xcode7.3
+      env: TARGET="Darwin 7.3"
```

Comment:
Use `TARGET=Darwin` for all of these, the xcode version shouldn't appear in the `TARGET` env var.